### PR TITLE
Severe audit of featured snaps

### PR DESCRIPTION
**Pull request by Lukewh**
# Done
- Removed any abandoned snaps
- Removed any snaps that have not received an update to <default>/stable in the last year

This is a big list of deletions, but is a necessary starting point to rebuild a set of genuinely useful and up-to-date snaps.

### CODE REVIEW

File: prod/development.section [hunk 1]
```diff
@@ -3,28 +3,23 @@ sublime-text
 pycharm-community
 vscode
```
Comment (sergiusens) [inline prod/development.section:4]: missing the most important of them all!
```suggestion
pycharm-community
emacs
vscode
```

File: prod/news-and-weather.section [hunk 2]
```diff
@@ -1,12 +1 @@
...
-halo-weather
-newsboat
-rssnotify
-rssepisodeextractor
-phoronix-news
-ubuntu-newsblur
-ubuntu-weather-app
-wethr
-yandex-weather-cli
 sunwait
```
Comment (sergiusens) [inline prod/news-and-weather.section:1]: why remove newsboat?

File: prod/photo-and-video.section [hunk 3]
```diff
@@ -24,6 +24,5 @@ mkvtoolnix-jz
 obs-studio
 smartscreen
 livetuner
-# orphans
 movian
 guvcview
```
Comment (sergiusens) [inline prod/photo-and-video.section:28]: snap I maintain, but can we add darktable and digikam here?

File: prod/news-and-weather.section (hunk 2 above)
Comment (Lukewh) [reply prod/news-and-weather.section]: Whoops, thanks!

File: prod/photo-and-video.section (hunk 3 above)
Comment (Lukewh) [reply prod/photo-and-video.section]: Done!

**Pull request merged**